docs/source/learn/examples.md

```diff
@@ -37,4 +37,5 @@ examples/pymc_overview
 examples/GLM_linear
 examples/model_comparison
 examples/posterior_predictive
+examples/shapes
```

Comment:
the file is called `dimensionality`, it is not being included in the docs correctly with this.